.github/workflows/build-release.yml

```diff
@@ -1,11 +1,12 @@
 name: build-release
 on:
   push:
+    branches:
+      - master
     paths:
       - gradle.properties
 jobs:
   deploy:
-    if: github.ref == 'refs/heads/master'
```

Comment:
Not sure if this will work correctly. The field `on` acts like a `OR`. So in this case I think it will trigger the release on any push to master.

Reply:
We have been using like this in PHP
https://github.com/checkout/checkout-sdk-php/blob/master/.github/workflows/build-release.yml

Also in Python as well